docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/03_LORIS_MRI_Install/Signal_Viewer_UI_Install.md

```diff
@@ -0,0 +1,277 @@
+
+
+# Signal Viewer UI Install
+
+> Last update: 2022-11-02
+
+## Context
+
+This document is a guide to install and configure Loris HBCD to use the `Signal Viewer` UI for EEG (BIDS files).
+
+## TOC
+
+1. Install
+    1. [Environment](#environment)
+    1. [Versions](#versions)
+    1. [Install Loris-MRI](#install-loris-mri)
+    1. [Install Protobuf](#install-protobuf)
+    1. [Update `electrophysiology_browser` module](#update-electrophysiology_browser-module)
+    1. [Installed environment](#installed-environment)
+1. [EEG upload to Loris](#eeg-upload-to-loris)
+    1. [Get EEG files](#get-eeg-files)
+    1. [Chunk EEG files](#chunk-eeg-files)
+    1. [Import BIDS to Loris](#import-bids-to-loris)
+1. [Troubleshooting](#troubleshooting)
+    1. [BIDS import](#bids-import)
+
+## Install
+
+### Environment
+
+> Note: right now, only `24.0 release` with `php 8.0` works. More recent versions are on the way.
+
+Environment:
+- `Ubuntu` machine
+- A `Loris instance` at `24.0` release (or `24.0-release` branch)
+- HBCD project should in `project` folder (`staging` branch)
+
+### Versions
+
+Versions:
+- `php 8.0`, and all packages required for php and apache:
+
+```bash
+# install
+sudo apt install -y \
+    php 8.0 \
+    php8.0-mysql \
+    php8.0-xml \
+    php8.0-mbstring \
+    php8.0-gd \
+    php8.0-zip \
+    php8.0-curl \
+    libapache2-mod-php8.0
+
+# select php8.0
+sudo update-alternatives --config php
+
+# in Loris, update php deps
+cd /path/to/loris
+composer update
+
+# make
+make
+make dev
+```
+
+- Apache2 with `php8.0` module enabled
+
+```bash
+# disable php modules in apache
+sudo a2dismod php7.4
+sudo a2dismod php8.1
+
+# enable apache php 8.0 module
+sudo a2enmod php8.0
+
+# restart apache
+sudo service apache2 restart
+```
+
+### Install Loris-MRI
+
+Follow the steps mentionned in [Loris-MRI README.md](https://github.com/regisoc/Loris-MRI#installation). This includes the [BIC MNI toolkit](https://github.com/aces/Loris-MRI#3-install-minc-toolkit-from-httpbic-mnigithubio) and [a Loris database update](https://github.com/aces/Loris-MRI#4-run-installer-to-set-up-directories-configure-environment-install-perl-libraries-and-dicom-toolkit).
+
+### Install Protobuf
+
+Install [Protobuf](https://github.com/protocolbuffers/protobuf#protocol-compiler-installation). See also [PR #8193](https://github.com/aces/Loris/pull/8193) to install a recent version of Protobuf with current troubleshooting.
+
+### Update `electrophysiology_browser` module
+
+Update the module with the [step detailed for visualization features](https://github.com/aces/Loris/tree/main/modules/electrophysiology_browser#-installation-requirements-to-use-the-visualization-features).
```

Comment:
Follow the [installation steps for the visualization features](https://github.com/aces/Loris/tree/main/modules/electrophysiology_browser#-installation-requirements-to-use-the-visualization-features).